src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,5 +61,25 @@ private void configureButtonBindings() {
             
     }
 
+    private void autoConfig() {
+        autoChooser.setDefaultOption("1- 1 Cube + Balance", "1-1_Cube_Balance");
+        autoChooser.addOption("1- 2 cube", "1-2_Cube_Balance");
+        autoChooser.addOption("3- 1 Cube + Balance", "3-1_Cube_Balance");
+        autoChooser.addOption("3- 2 Cube", "3-2_Cube");
+        SmartDashboard.putData("Auto Mode", autoChooser);
+
+    }
+
+    public Command getAutonomousCommand() {
+        if (activeAutoCommand == null || !activeAutoName.equals(autoChooser.getSelected())) {
+            activeAutoCommand = new AutoRoutines(autoChooser.getSelected(), arm, new DriveSubsystem()); //TODO: switch to actual drive subystem
```

Comment:
putting a comment here so we remember to do this

Resolve when the actual drive subsystem is used